paperqa/clients/__init__.py

```diff
@@ -201,7 +201,7 @@ async def upgrade_doc_to_doc_details(self, doc: Doc, **kwargs) -> DocDetails:
         # note we have some extra fields which may have come from reading the doc text,
         # but aren't in the doc object, we add them here too.
         extra_fields = {
-            k: v for k, v in kwargs.items() if k in {"title", "authors", "doi"}
+            k: v for k, v in kwargs.items() if k in set(DocDetails.model_fields)
         }
```

Comment:
@mskarlin this was necessary for the dockey to be equal to doc_id.
Do you know why it was like that previously and if its ok to change it?